Sources/PT.PM.Common/MessagePack/RootUstMessagePackSerializer.cs

```diff
@@ -20,6 +20,9 @@ public class RootUstMessagePackSerializer
         private TextSpanSerializer textSpanSerializer;
         private FileSerializer fileSerializer;
 
+        // One file can contain several RootUst nodes. For example, the file with PHP inside JavaScript inside PHP
+        // Deserialization is being occurred in depth-first manner, that's why it's possible to fill roots and parents
```

Comment:
> Deserialization is being occurred in depth-first manner

"Deserialization is (being) conducted in depth-first manner" maybe? Or "performed"?

And if we first visit parent and then descendant (`RootUst`s) that's not "depth-first". Depth-first is bottom-up and in this case we wouldn't have a reference to parent `RootUst` when we process descendant. So it's a "top-down" manner, I think.

Reply:
You are right: the "top-down" term is more correct.